man/crypttab.xml

```diff
@@ -260,6 +260,29 @@
         <xi:include href="version-info.xml" xpointer="v243"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>link-volume-key-to-keyring=</option></term>
+
+        <listitem><para>Specifies the keyring and key description where
+        LUKS2 volume key gets linked during device activation. The keyring
```

Comment:
hmm, this only links the LUKS2 volume key? any reason why this doesn't covered LUKS1 vol keys too? or others?

Reply:
Correct. The idea was "do not touch LUKS1 code unless there's a bug." But I can take a look how invasive it would be to support it.

Reply:
Oh, i don#t really care about luks1. I was just wondering, because you wrote this so specifically.

I'd just drop the "LUKS2" volume key frankly. Even if this is currently only supported by LUKS2, the concept is generic. I'd just say we link the volume key in. That gives libcryptsetup to add this feature one day or not.

As long as there's a reasonable error message when we try this on non-LUKS2 for now I am happy.

Hence, don#t bother with adding this to luks1, not for me.